tests/functional/pod_and_daemons/test_csiaddon_pod_security.py

```
@@ -42,45 +42,53 @@ def test_csi_addon_pod_security(self):
                 container_name="csi-addons",
                 out_yaml_format=False,
             )
-            log.info(
+            logger.info(
                 f"CSI Addon pod is reachable securely on port {port_used_by_csi_addon}"
             )
         except CommandFailed as e:
-            log.error(
+            logger.error(
                 f"CSI Addon pod is not reachable securely on port {port_used_by_csi_addon}: {str(e)}"
             )
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)